source/openapi3/split.yaml

```diff
@@ -2010,6 +2010,29 @@ paths:
                 $ref: '#/components/schemas/UpdateAContactResponse'
         '404':
           description: Not Found
+  '/contacts/{id}/refresh_balance':
+    post:
+      tags:
+        - Contacts
+      summary: Refresh contact bank connection
+      description: Request the bank connection for a contact to refresh available funds. This is intended for use in conjuction with the [Payment Request](/#request-payment) `precheck_funds` option.
```

Comment:
Consider suggesting that this be ran atleast 60 seconds before the PR is expected to be made.